lecture-01/slices/main.go

```diff
@@ -107,3 +149,65 @@ func mergeUniqueInts(slices ...[]int) []int {
 
 	return res
 }
+
+// removeSameInts removes elements from 'dest' slice when they exist in 'source' slice
+func removeSameInts(dest, source []int) []int {
+	m := make(map[int]bool)
+
+	for _, val := range source {
+		m[val] = true
+	}
+
+	for i := 0; i < len(dest); i++ {
```

Comment:
Do you need that loop or it's possible just do something with the map?

Reply:
hint? 👀